prow/manifests/overlays/metal3/config.yaml

```diff
@@ -2527,7 +2508,7 @@ presubmits:
     optional: false
   - name: metal3-centos-e2e-integration-test-release-1-7
     branches:
-    - release-24.0
+    - release-24.1
```

Comment:
Here we need to fix gh Branch protection as well since all releases required main tests there